pkg/internal/ebpf/common/spanner.go

```diff
@@ -58,7 +65,25 @@ func HTTPRequestTraceToSpan(trace *HTTPRequestTrace) request.Span {
 			UserPID:   trace.Pid.UserPid,
 			Namespace: trace.Pid.Ns,
 		},
+		ErrorMessage:    string(trace.Error.ErrMsg[:errMsgLen]),
+		ErrorStacktrace: extractErrorStacktrace(trace, filter.GetSymTab(trace.Pid.UserPid)),
+	}
+}
+
+func extractErrorStacktrace(trace *HTTPRequestTrace, symTab *gosym.Table) string {
+	var stacktrace strings.Builder
+	if symTab != nil && trace.Error.UstackSz > 0 {
+		for _, pc := range trace.Error.Ustack {
+			f := symTab.PCToFunc(pc)
```

Comment:
I think we need to check how this works with multiple processes launched from the same executable. I'm wondering if the symbol table will be meant for the address range of the initial executable which launched. What I mean by this is that I think you must know the executable base memory address to be able to do the PC to symbol translation, because technically the symbol table has only offsets from the executable start, but the PC that comes from eBPF is the absolute memory address of the instruction pointer.

Let's take the following example:

1. Imagine we have a symbol table with one function `foo()` at offset `0x100`.
2. We launch the executable once and its base code virtual address is `0x20000`.
3. We launch the executable one more time and this one starts at virtual address `0x30000`.
4. We get exception at 0x200105 and then at 0x300105. Technically that's exception at two different addresses, but it's the same exception in two different processes. 

We'll only parse the symbol table once, since we detect that the executable is already instrumented.

I think this code translation must take into account the base memory address of the executable, I think the symbol table is not sufficient.